pennylane/tape/qscript.py

```diff
@@ -1369,6 +1382,15 @@ def from_queue(cls, queue, shots: Optional[Union[int, Sequence, Shots]] = None):
         return cls(*process_queue(queue), shots=shots)
 
 
+def _warn_name():
+    warnings.warn(
+        "The ``name`` property and keyword argument of ``QuantumScript`` is deprecated and will be"
+        " removed in the next release. Going forward, please refrain from using them. This also affects"
+        " the ``QuantumTape`` and ``OperationRecorder`` classes.",
+        UserWarning,
+    )
+
+
```

Comment:
Let's place this at the top of the file instead of the bottom. Just some personal preference to define private utilities above the things that use them.

Reply:
Done!